Mangled output in Copr logs

Observing Copr logs, there is corrupted output, presumably due to some encoding issues. Looking e.g. at [libsbml build](https://copr.fedorainfracloud.org/coprs/vondruch/mpb.checker/build/6771727/), the output contains:

~~~
/builddir/build/BUILD/libsbml-5.20.2/src/sbml/compress/zipfstream.cpp: In function â€˜void* zipopen(const char*, const char*, int)â€™:
/builddir/build/BUILD/libsbml-5.20.2/src/sbml/compress/zipfstream.cpp:545:41: error: cannot convert â€˜long unsigned int*â€™ to â€˜uint32_t*â€™ {aka â€˜unsigned int*â€™}
  545 |     filetime(filenameinzip,&zi.tmz_date,&zi.dosDate);
      |                                         ^~~~~~~~~~~
      |                                         |
      |                                         long unsigned int*
/builddir/build/BUILD/libsbml-5.20.2/src/sbml/compress/zipfstream.cpp:97:64: note:   initializing argument 3 of â€˜long unsigned int filetime(const char*, tm_zip*, uint32_t*)â€™
   97 | unsigned long filetime(const char* f, tm_zip* tmzip, uint32_t* dt);
      |                                                      ~~~~~~~~~~^~
~~~

Please note the `â€™` sequences. Comparing with [Koji](https://koji.fedoraproject.org/koji/taskinfo?taskID=110556201), where the output is fine:

~~~
/builddir/build/BUILD/libsbml-5.20.2/src/sbml/compress/zipfstream.cpp: In function ‘void* zipopen(const char*, const char*, int)’:
/builddir/build/BUILD/libsbml-5.20.2/src/sbml/compress/zipfstream.cpp:545:41: error: cannot convert ‘long unsigned int*’ to ‘uint32_t*’ {aka ‘unsigned int*’}
  545 |     filetime(filenameinzip,&zi.tmz_date,&zi.dosDate);
      |                                         ^~~~~~~~~~~
      |                                         |
      |                                         long unsigned int*
/builddir/build/BUILD/libsbml-5.20.2/src/sbml/compress/zipfstream.cpp:97:64: note:   initializing argument 3 of ‘long unsigned int filetime(const char*, tm_zip*, uint32_t*)’
   97 | unsigned long filetime(const char* f, tm_zip* tmzip, uint32_t* dt);
      |                                                      ~~~~~~~~~~^~
~~~

I know that with RPM 4.19 the default encoding changed from `C` to `C.UTF-8`. Can it be the reason for the issue?

P.S. sorry if the links to the logs disappeared by the time you are looking at the ticket.

--- a/backend/conf/lighttpd/mime.conf
+++ b/backend/conf/lighttpd/mime.conf
@@ -52,7 +52,7 @@ mimetype.assign             = (
   ".conf"         =>      "text/plain",
   ".text"         =>      "text/plain",
   ".txt"          =>      "text/plain",
-  ".spec"         =>      "text/plain",
+  ".spec"         =>      "text/plain; charset=utf-8",
   ".dtd"          =>      "text/xml",
   ".xml"          =>      "text/xml",
   ".mpeg"         =>      "video/mpeg",

